### ISSUE TITLE
Inconsistent build scripts

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#39507]
build/build.php contains three lists of files/folders not to include in a release

1. Removes .DS_Store; .git sources; testing, CI, and IDE configuration files; Changelogs; GitHub Meta; and README files
2. Here we set the files/folders which should not be packaged at any time
3. Here we set the files/folders which should not be packaged with patch packages only

List one does not include all the files per the description eg .drone.yml
List two includes most of the files again from list one plus some that should have been in list one eg .drone.yml
List 3 is ok

In summary list one is missing some files and list two is trying to remove files that will never be there because they were already removed by list one

Using the three lists incosistently like this not only leads to duplication but is harder to maintain and has the potential to accidentally include a file that should not be distributed

**Comment by brianteeman** [comment]
closed due to lack of interest
